docs/status.md

```diff
@@ -53,7 +53,7 @@ one of the following tracking labels.
 | compare                  |         no         |        yes*        |        yes*         |         yes          |        no        |
 | complex                  |         no         |        yes*        |        yes*         |         yes          |        no        |
 | compute_reshape_shape    |         no         |         no         |         no          |         yes          |        no        |
-| concatenate              |         no         |        yes*        |        yes*         |         yes          |        no        |
+| concatenate              |        yes         |        yes*        |        yes*         |         yes          |        no        |
```

Comment:
What about verification and type inference? Do they conform to the newly updated spec?

Reply:
No. Currently, verify does not check that size of `inputs` is at least one:
* (C3) `inputs` have N tensors where N >= 1.

Reply:
Looks like the verifier does check it actually, https://github.com/openxla/stablehlo/blob/4859634f3d95851a7a55c16cf9d753327a57c860/stablehlo/dialect/StablehloOps.cpp#L3022
so I'll mark it as `yes`.